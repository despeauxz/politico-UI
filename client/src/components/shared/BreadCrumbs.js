import React from 'react';
import { Link } from 'react-router-dom';
import PropTypes from 'prop-types';


const BreadCrumbs = (props) => {
    return (
        <nav className="bg-grey-light rounded font-sans w-full flex mb-4 items-start text-sm">
            <ol className="list-reset flex text-grey-dark">
                <li>
                    <Link to="#" className="text-black font-bold">Home</Link>
                </li>
                {props.children}
            </ol>
        </nav>
    );
};

BreadCrumbs.propTypes = {
    children: PropTypes.object
};

export default BreadCrumbs;
