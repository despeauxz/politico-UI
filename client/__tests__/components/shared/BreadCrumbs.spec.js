import React from 'react';
import BreadCrumbs from '@components/shared/BreadCrumbs';


describe('<BreadCrumbs />', () => {
    it('render the breadcrumbs component', () => {
        const shallowWrapper = shallow(<BreadCrumbs>
            <h3>Hello World</h3>
        </BreadCrumbs>);

        expect(toJson(shallowWrapper)).toMatchSnapshot();
        expect(shallowWrapper.find('BreadCrumbs')).toBeTruthy();
    });
});
