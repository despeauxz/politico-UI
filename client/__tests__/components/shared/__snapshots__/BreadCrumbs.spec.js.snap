// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BreadCrumbs /> render the breadcrumbs component 1`] = `
<nav
  className="bg-grey-light rounded font-sans w-full flex mb-4 items-start text-sm"
>
  <ol
    className="list-reset flex text-grey-dark"
  >
    <li>
      <Link
        className="text-black font-bold"
        to="#"
      >
        Home
      </Link>
    </li>
    <h3>
      Hello World
    </h3>
  </ol>
</nav>
`;
